docs/operations/automatic-data-locality.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Automatic data locality"
+linkTitle: "Automatic data locality"
+---
+
+## Ondat volumes move with their pods
+
+Having an Ondat volume instance on the same node as the pod using it can bring some benifits to your application. Unfortunatly pods can move around over time leaving the Volume behind.
```

Comment:
It would be good to specify what the benefits are, which is basically it's more performant

Reply:
Typos:
```suggestion
Having an Ondat volume instance on the same node as the pod using it can bring some benefits to your application. Unfortunately pods can move around over time leaving the volume on a different node.
```